src/Confluent.Kafka/Config.cs

```diff
@@ -472,19 +472,29 @@ protected void SetObject(string name, object val)
         public BrokerAddressFamilyType? BrokerAddressFamily { get { return (BrokerAddressFamilyType?)GetEnum(typeof(BrokerAddressFamilyType), "broker.address.family"); } set { this.SetObject("broker.address.family", value); } }
 
         /// <summary>
-        ///     Throttle broker reconnection attempts by this value +-50%.
+        ///     When enabled the client will only connect to brokers it needs to communicate with. When disabled the client will maintain connections to all brokers in the cluster.
+        /// </summary>
+        public bool? EnableSparseConnections { get { return GetBool("enable.sparse.connections"); } set { this.SetObject("enable.sparse.connections", value); } }
+
+        /// <summary>
+        ///     **DEPRECATED** No longer used. See `reconnect.backoff.ms` and `reconnect.backoff.max.ms`.
         /// </summary>
         public int? ReconnectBackoffJitterMs { get { return GetInt("reconnect.backoff.jitter.ms"); } set { this.SetObject("reconnect.backoff.jitter.ms", value); } }
 
         /// <summary>
-        ///     librdkafka statistics emit interval. The application also needs to register a stats callback using `rd_kafka_conf_set_stats_cb()`. The granularity is 1000ms. A value of 0 disables statistics.
+        ///     The initial time to wait before reconnecting to a broker after the connection has been closed. The time is increased exponentially until `reconnect.backoff.max.ms` is reached. -25% to +50% jitter is applied to each reconnect backoff. A value of 0 disables the backoff and reconnects immediately.
         /// </summary>
-        public int? StatisticsIntervalMs { get { return GetInt("statistics.interval.ms"); } set { this.SetObject("statistics.interval.ms", value); } }
+        public int? ReconnectBackoffMs { get { return GetInt("reconnect.backoff.ms"); } set { this.SetObject("reconnect.backoff.ms", value); } }
+
+        /// <summary>
+        ///     The maximum time to wait before reconnecting to a broker after the connection has been closed.
+        /// </summary>
+        public int? ReconnectBackoffMaxMs { get { return GetInt("reconnect.backoff.max.ms"); } set { this.SetObject("reconnect.backoff.max.ms", value); } }
 
         /// <summary>
-        ///     Logging level (syslog(3) levels)
+        ///     librdkafka statistics emit interval. The application also needs to register a stats callback using `rd_kafka_conf_set_stats_cb()`. The granularity is 1000ms. A value of 0 disables statistics.
         /// </summary>
-        public int? Log_Level { get { return GetInt("log_level"); } set { this.SetObject("log_level", value); } }
```

Comment:
Should this have been removed?  It still exists in the [librdkafka documentation](https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md).

Reply:
yeah, I noticed that. I assumed it was for good reason (this is a useless property in practice, and therefore confusing - use `debug` instead), but don't quickly see why. will investigate.